WPFWijkagent/WPFWijkagent/WitnessController.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Text;
+using Tweetinvi;
+using WijkagentModels;
+
+namespace WijkagentWPF
+{
+    public class WitnessController
+    {
+        public Offence _Offence { get; set; }
+        private AskForWitnessWindow witnessWindow;
+
+        public WitnessController(Offence offence, MainWindow window)
+        {
+            witnessWindow = new AskForWitnessWindow(this, window);
```

Comment:
steven: eerst het scherm dan de controller niet andersom en die mainwindow is ook niet nodig.